### ISSUE TITLE
support multiple client connections

### ISSUE THREAD

**Issue by mossberg** [mossberg/poet#65]
if you're already using poet and a client connects put a notification on the screen and provide a command to view current clients and switch between them. have the prompt reflect which one you're on via number and maybe hostname/os or something

**Comment by themazzaroth** [comment]
Any progress on this feature?

**Comment by mossberg** [comment]
Nope, sorry :) Thanks for letting me know you’re interested in this feature though, I’ll prioritize it!
